tools/admin/permit-holders/doctor-information.ts

```diff
@@ -9,7 +9,7 @@ import {
 export type DoctorFormData = Pick<
   Physician,
   'firstName' | 'lastName' | 'phone' | 'addressLine1' | 'addressLine2' | 'city' | 'postalCode'
-> & { mspNumber: number | null };
+> & { mspNumber: string | null };
```

Comment:
```suggestion
| 'mspNumber' > ;
```

Same as previous comment about mspNumber type, we don't need this field to be nullable so we can take it directly from the `Physician` object.